aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java

```diff
@@ -307,15 +428,22 @@ public void run(String... args) {
                 Period elapsedPeriod = new Period(startTime, stopTime);
                 String elapsedTimeString = formatPeriodString(elapsedPeriod);
 
+                //  Log all job details including statistics on how long it took.
                 logger.info("Aggregation completed successfully. JobID [" + batchJobId + "], Callback email [" + contactEmail + "], Size bytes [" + convertedFile.toFile().length() + "], Elapsed time (h:m:s) [" + elapsedTimeString + "]");
                 statusDocument = statusBuilder.createResponseDocument(EnumStatus.SUCCEEDED, GOGODUCK_PROCESS_IDENTIFIER, null, null, outputMap);
                 statusFileManager.write(statusDocument, statusFilename, STATUS_FILE_MIME_TYPE);
 
+
                 //  Send email - if email address was provided
                 if (contactEmail != null) {
                     try {
                         //  Send completed job email to user
-                        emailService.sendCompletedJobEmail(contactEmail, batchJobId, statusUrl, S3Utils.getExpirationinDays(outputBucketName));
+                        emailService.sendCompletedJobEmail(
+                                contactEmail,
+                                batchJobId,
+                                statusUrl,
+                                S3Utils.getExpirationinDays(outputBucketName),
+                                EmailService.formatRequestDetail(subsetParams, collectionTitle));
```

Comment:
Why doesn't the email service do this there?  Its kind of complicated to have to call the EmailService to format parameters to be passed to the email service.

Reply:
Done